test/db/anal/x86_64

```diff
@@ -3348,6 +3336,33 @@ EXPECT=<<EOF
 EOF
 RUN
 
+NAME=test_switch_indirect.exe switch/case flags
```

Comment:
nice!

Reply:
only thing is that IDA is able to recognize that ebx comes from `ecx - 1` so all the cases are actually +1. But I'd say this is already a very nice improvement :)